src/v/redpanda/admin/server.cc

```diff
@@ -1860,6 +1863,21 @@ admin_server::patch_cluster_config_handler(
                       yaml_name,
                       property.format_raw(yaml_value),
                       validation_err.value().error_message());
+                } else if (auto restricted_err = property.check_restricted(val);
+                           restricted_err.has_value() && should_sanction) {
+                    // TODO(oren): prob better to format the whole message like
+                    // the log?
```

Comment:
Isn't that what you're doing already by reusing the `error_message`?

Reply:
done